example/App.js

```diff
@@ -200,6 +201,30 @@ const App = () => {
           forceTitle
         />
       </div>
+      <Heading type="sub">Support for addons</Heading>
+      <Description>
```

Comment:
Can you please add a subsection named Available Addons, and list the addons with a small description.